.github/workflows/version.hutch-manager.yml

```diff
@@ -0,0 +1,68 @@
+name: Bump Manager Version
+
+on:
+  workflow_dispatch: # manual trigger
+    inputs:
+      version_tag:
+        description: Version tag
+        required: true
+  push:
+    branches:
+      - release/manager-**
+
+env:
+  CI_build-config: release
+  CI_dotnet-version: 6.0.x
+  CI_node-version: "16"
+  CI_publish-dir: publish
+
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+  DOTNET_NOLOGO: true
+
+jobs:
+  get-version:
+    runs-on: ubuntu-latest
+    outputs:
+      VERSION_TAG: ${{ steps.get-tag-auto.outputs.VERSION_TAG || steps.get-tag-manual.outputs.VERSION_TAG}}
+    steps:
+      - name: Get Version Tag
+        id: get-tag-auto
+        if: ${{ input.version_tag }} == ''
+        run: |
+          echo ::set-output name=VERSION_TAG::$(echo ${GITHUB_REF_NAME#release/manager-})
+      - name: Get Version Tag (manual)
+        id: get-tag-manual
+        if: ${{ input.version_tag }} != ''
+        run: |
+          echo ::set-output name=VERSION_TAG::${{ input.version_tag }}
+
+  bump-version:
+    needs: get-version
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+      - uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 6.30.0
```

Comment:
this should match `package.json` version of pnpm.

While we're here though, update the `package.json` version to the latest (`7.13.4`)